src/java/org/apache/cassandra/config/Config.java

```diff
@@ -482,6 +482,11 @@
 
     public volatile boolean diagnostic_events_enabled = false;
 
+    // Default and minimum keyspace replication factors allow validation of newly created keyspaces
+    // and good defaults if no replication factor is provided by the user
+    public int default_keyspace_rf = 1;
```

Comment:
Should this be volatile? I'm not sure, but I have a suspicion that all fields supposed to be updated are marked `volatile`. I hope someone from project veterans could confirm the assumption.